composer.json

```diff
@@ -13,6 +17,21 @@
         "symfony/var-dumper": "^5.4",
         "vimeo/psalm": "^5.9"
     },
+    "config": {
+        "platform": {
+            "php": "8.0.27"
```

Comment:
You defined in `composer.json` as min PHP version the 8.0.2, you cannot define here a higher version 🤔